pkg/master/master.go

```diff
@@ -392,12 +390,6 @@ func DefaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
 		policyapiv1beta1.SchemeGroupVersion,
 		rbacv1.SchemeGroupVersion,
 		rbacv1beta1.SchemeGroupVersion,
-		// Don't copy this pattern. We enable rbac/v1alpha1 and settings/v1laph1
-		// by default only because they were enabled in previous releases.
-		// See https://github.com/kubernetes/kubernetes/pull/47690.
-		// TODO: disable rbac/v1alpha1 and settings/v1alpha1 by default in 1.8
-		rbacv1alpha1.SchemeGroupVersion,
-		settingv1alpha1.SchemeGroupVersion,
 		schedulingapiv1alpha1.SchemeGroupVersion,
```

Comment:
Eh, the schduelingv1alpha1 is enabled by default. It's introduced in 1.8, so we can just remove it from the list. @jennybuckley could you remove this line as well?

cc @bsalamat the original author.

Reply:
Yes, I removed it as well as from the build file and the imports.
I was going to bring it up because I noticed that there was another alpha feature enabled by default, but I wasn't sure if that was intentional.